src/content/pages/sponsors.html

```diff
@@ -25,6 +25,35 @@
 }) }}
 
 <section class="section">
+
+    {% if sponsors.diamond.length %}
+    <div class="content-container grid grid--center-items">
+        <div class="span-1-5">
+            <h2 class="section-title h4" id="diamond">{{ site.sponsors_page.tiers.diamond.title }}</h2>
+            <p>
+                {{ site.sponsors_page.tiers.diamond.description }}
+            </p>
+        </div>
+        <div class="span-7-12">
+            <ul role="list" class="sponsors sponsors--diamond">
+                {% for sponsor in sponsors.diamond %}
+                    {% set sponsor_amount = sponsor.monthlyDonation %}
+                    {% set sponsor_title = site.sponsors_page.sponsor_title | replace("SPONSOR_NAME", sponsor.name) | replace("AMOUNT", sponsor_amount | dollars) %}
+
+                <li>
+                    <a href="{{ sponsor.url | url }}" class="sponsor-link" rel="nofollow noopener sponsored" target="_blank">
+                        <img loading="lazy" decoding="async" src="{{ sponsor.image }}" height="128"
```

Comment:
```suggestion
                        <img loading="lazy" decoding="async" src="{{ sponsor.image }}" height="160"
```

For the diamond, we should set the height to 160px to ensure it appears larger than platinum. However, in the code, platinum, gold, silver, and bronze all have the same height of 128px. Was this height intentionally set to 128px for all?

Currently, the height attribute has no impact because styles are applied through class names, so we don't notice any issues in the UI.

Additionally, I checked the styles, and both diamond and platinum have the same maximum height. Does this mean we will display the images of diamond and platinum sponsors at the same size?
```
.sponsors.sponsors--platinum,
.sponsors.sponsors--diamond {
	picture,
	img {
		max-height: 8rem;
	}
}

.sponsors.sponsors--gold {
	picture,
	img {
		max-height: 6rem;
	}
}

.sponsors.sponsors--silver {
	picture,
	img {
		max-height: 4rem;
	}
}

.sponsors.sponsors--bronze,
.sponsors.sponsors--technology {
	picture,
	img {
		max-height: 3rem;
	}
}
```

Reply:
Yes, the intent is that 128 is the largest that we do. Any bigger than that and things start looking strange.

Reply:
are we doing 128 for both mobile and desktop screen?

Reply:
yes